en/docs/references/alerting.md

```diff
@@ -0,0 +1,61 @@
+
+![Diagnostic view](../assets/img/alerting/diagnostic-view-values.png){.cInlineImage-half}
+
+When you hover over the graph, you can see the Memory usage and it growing steadily and reaching the maximum value. This will cause the application to restart and it will log the Out of Memory error.
+
+#### Selecting custom time ranges
+
+If the time range that is selected automatically for you is not enough, you can select the time range that you desire and debug further. You can do this by clicking on the Time selection drop down bar shown by 1.
+
+![Custom Time range](../assets/img/alerting/custom-time-range.png){.cInlineImage-half}
+
+Then you can select the **Custom** option, this will automatically add the time range that is already given by the alert email. You can modify them to see the **Diagnostic View** for that  specific time range.
+
+For more information on how to use the Observability view for Root cause analysis check out the following [page](https://wso2.com/choreo/docs/observability/root-cause-analysis/). 
+
+### Application Error Alert
+
+This alert is triggered when you use the **“log:printError()”** function in your component and it logs an error. We consider that your component must have faced some serious problem and send you an email to notify you about this. The sample email will look like this. The email will provide details about the application and the **Check in Portal** link to view this log along with the request that caused this. You can also get support from WSO2 to troubleshoot this issue by using the **Contact us** link provided in the email.
+
+![Application Error email](../assets/img/alerting/application-error-email.png){.cInlineImage-full}
+
+#### How to troubleshoot Application error in the Observability view
+
+Once you click on the **Check in Portal** link you will be redirected to the observe page of the component in the Choreo console
+
+![Application Error Troubleshooting](../assets/img/alerting/application-error-troubleshooting.png){.cInlineImage-full}
+
+This will open the **Throughput and Latency** view and automatically select the request that caused this error log. This is shown in **1**, it will show you the throughput and latency of the particular request. The color of the graph will tell you whether that request is successful or an error. You can also see the status of the request; its latency and status code indicated in a green colored circle in **4**. You will also see the error log that triggered the alert email in **2**. The time range used to highlight the relevant error log is indicated in **3**.  But you can scroll up and down and get more logs within the time range specified in **5**. You can also diagnose further by changing the selected time ranges. See the (**Selecting custom time ranges**) section above.
```

Comment:
**Throughput and Latency** should be called as `view` because it is a small part of the whole webpage whereas `tab` usually means the whole webpage itself

Reply:
"view" is ok.